### DISCUSSION TITLE
Welcome to express-typescript Discussions!

### DISCUSSION THREAD

**Discussion by faizahmedfarooqui** [GeekyAnts/express-typescript#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by trupti28** [comment]
How do I configure the mongodb and dump the database?

**Reply by faizahmedfarooqui** [reply]
You can set your MongoDB configurations in the [**.env**](https://github.com/GeekyAnts/express-typescript/blob/master/.env#L58) (_click on the link to see the line to configure in the file_) file.
